starters/adapters/aws-lambda/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "description": "AWS Lambda",
   "scripts": {
-    "build.server": "vite build -c adapters/aws-lambda/vite.config.ts",
+    "build.server": "qwik check-client dist/q-manifest.json && vite build -c adapters/aws-lambda/vite.config.ts",
```

Comment:
so this should become `qwik check-client src dist`. The first is the location of the client source code and the second is the client output directory. The q-manifest.json is always the same.